src/full/Agda/Utils/Maybe.hs

```diff
@@ -23,12 +23,12 @@ boolToMaybe b x = if b then Just x else Nothing
 
 -- * Collection operations.
 
--- | @unionWith@ for collections of size <= 1.
-unionMaybeWith :: (a -> a -> a) -> Maybe a -> Maybe a -> Maybe a
-unionMaybeWith f Nothing mb      = mb
-unionMaybeWith f ma      Nothing = ma
-unionMaybeWith f (Just a) (Just b) = Just $ f a b
-
+---- | @unionWith@ for collections of size <= 1.
```

Comment:
Functions that are not used any more such as this one I usually prefix with a comment
```haskell
-- UNUSED <name> <date>
```
and comment them out.  Then if it has remained unused for some years it can be deleted.

Reply:
Done.